api/base/serializers.py

```diff
@@ -1039,7 +1061,9 @@ def to_representation(self, obj, envelope='data'):
         data = {
             'id': '',
             'type': type_,
-            'attributes': {},
+            'attributes': {
+                'social': {},
```

Comment:
Is this necessary? I would have expected, if you don't explicitly mention the social object, it would naturally be put in attributes. Please check.